pkg/builder/example_test.go

```diff
@@ -155,8 +155,3 @@ func (a *ReplicaSetReconciler) Reconcile(ctx context.Context, req reconcile.Requ
 
 	return reconcile.Result{}, nil
 }
-
-func (a *ReplicaSetReconciler) InjectClient(c client.Client) error {
```

Comment:
Same here, please update to set the client so the example will actually work (Unrelated to this change, it would be good if we tested that)